Assets/Scripts/NPCs/Vision.cs

```diff
@@ -1,88 +1,88 @@
 	}
 
 
 	//void DrawMesh()
 	//{
 	//	//int stepcount = Mathf.RoundToInt (viewAngle * meshresolution);
-	//	visibleTargets.Clear();
-	//	float stepAngleSize = viewAngle / (numVertex + 1);
-	//	List<Vector3> viewPoints = new List<Vector3>();
-
+	//	visibleTargets.Clear();
+	//	float stepAngleSize = viewAngle / (numVertex + 1);
+	//	List<Vector3> viewPoints = new List<Vector3>();
+
 	//	for (int i = 0; i < numVertex; i++)
-	//	{
-	//		RaycastHit2D hit;
-	//		Vector3 dir = DirFromAngle(angle + 90 + (viewAngle - (i + 1) * stepAngleSize * 2) / -2, false) * range * wayFacing.x;
-
+	//	{
+	//		RaycastHit2D hit;
+	//		Vector3 dir = DirFromAngle(angle + 90 + (viewAngle - (i + 1) * stepAngleSize * 2) / -2, false) * range * wayFacing.x;
+
 	//		if (!(hit = Physics2D.Raycast(transform.position, dir, range, obstacleMask)))
 	//		{
 	//			//	Debug.DrawRay (transform.position, dir, Color.red);
-	//			viewPoints.Add(dir);
+	//			viewPoints.Add(dir);
 	//		}
 	//		else
 	//		{
 	//			//	Debug.DrawRay (transform.position, DirFromAngle (angle + 90+(viewAngle - (i + 1) * stepAngleSize * 2) / -2, false) * wayFacing.x * hit.distance, Color.red);
-	//			viewPoints.Add(DirFromAngle(angle + 90 + (viewAngle - (i + 1) * stepAngleSize * 2) / -2, false) * wayFacing.x * hit.distance);
+	//			viewPoints.Add(DirFromAngle(angle + 90 + (viewAngle - (i + 1) * stepAngleSize * 2) / -2, false) * wayFacing.x * hit.distance);
 	//			if (hit.transform.name == "Player" && !GAME.Player.GetComponent<PlayerMovement>().crouched)
-	//			{
+	//			{
 	//				if (!visibleTargets.Contains(hit.transform))
-	//				{
-	//					visibleTargets.Add(hit.transform);
-	//				}
-	//			}
-	//		}
-	//	}
-
-	//	int vertexCount = viewPoints.Count + 1;
-	//	Vector3[] vertices = new Vector3[vertexCount];
-	//	int[] triangles = new int[(vertexCount - 2) * 3];
-
-	//	vertices[0] = Vector3.zero;
+	//				{
+	//					visibleTargets.Add(hit.transform);
+	//				}
+	//			}
+	//		}
+	//	}
+
+	//	int vertexCount = viewPoints.Count + 1;
+	//	Vector3[] vertices = new Vector3[vertexCount];
+	//	int[] triangles = new int[(vertexCount - 2) * 3];
+
+	//	vertices[0] = Vector3.zero;
 	//	for (int i = 0; i < vertexCount - 1; i++)
-	//	{
-
-
-	//		vertices[i + 1] = new Vector3(viewPoints[i].x, viewPoints[i].y, 0);
-
+	//	{
+
+
+	//		vertices[i + 1] = new Vector3(viewPoints[i].x, viewPoints[i].y, 0);
+
 	//		if (i < vertexCount - 2)
-	//		{
-	//			triangles[i * 3] = 0;
-	//			triangles[i * 3 + 1] = i + 1;
-	//			triangles[i * 3 + 2] = i + 2;
-	//		}
-	//	}
-
-	//	viewMesh.Clear();
-	//	viewMesh.vertices = vertices;
-	//	viewMesh.triangles = triangles;
-	//	viewMesh.RecalculateBounds();
+	//		{
+	//			triangles[i * 3] = 0;
+	//			triangles[i * 3 + 1] = i + 1;
+	//			triangles[i * 3 + 2] = i + 2;
+	//		}
+	//	}
+
+	//	viewMesh.Clear();
+	//	viewMesh.vertices = vertices;
+	//	viewMesh.triangles = triangles;
+	//	viewMesh.RecalculateBounds();
 	//	viewMesh.RecalculateNormals();
-	//}
-}
+	//}
+}
```

Comment:
I think we can just delete this file. I started making this during game jam 2 but then put everything in FieldOfVision instead.